lnrpc/wtclientrpc/wtclient.go

```diff
@@ -306,16 +333,42 @@ func (c *WatchtowerClient) GetTowerInfo(ctx context.Context,
 		return nil, err
 	}
 
+	var (
```

Comment:
That's quite a bit of duplicated code (line 273)

Reply:
indeed. have moved it into a helper function instead 👍